quickstart/src/selectroom.js

```diff
@@ -28,7 +28,7 @@ function selectRoom($modal, error) {
   }
 
   if (error) {
-    $alert.text(getUserFriendlyError(error));
+    $alert.html(getUserFriendlyError(error));
```

Comment:
For the error displayed in the room selection modal. Please add the title and share a screenshot.

Reply:
Sounds good, here it is : 
![image](https://user-images.githubusercontent.com/43423318/87484185-683bf380-c5ea-11ea-8b1f-8b38a5167319.png)

Reply:
@manjeshbhargav Overconstrained error doesn't seem to have a message but I've added the logic needed to display the message if there is one.
![image](https://user-images.githubusercontent.com/43423318/87488084-43e51480-c5f4-11ea-9881-5d6b5b22926d.png)

Reply:
@PikaJoyce I agree. Let's keep unbolded.